lib/astarte_core/cql_utils.ex

```diff
@@ -167,4 +167,40 @@ defmodule Astarte.Core.CQLUtils do
 
     eid
   end
+
+  @spec get_from_realm!(any()) :: any()
+  def get_from_realm!("") do
+    raise "EmptyRealm"
+  end
+
+  def get_from_realm!(realm_name) when is_binary(realm_name) do
```

Comment:
I would go with 
```suggestion
  @spec realm_name_to_keyspace_name(nonempty_binary()) :: nonempty_binary()
  def realm_name_to_keyspace_name(realm_name) do
```
So that we keep consistency in naming and behavior with other functions in the module.

Another option, since the instance id will be known by the user of the library,  could be
```elixir
@spec realm_name_to_keyspace_name(nonempty_binary()) :: nonempty_binary()
@spec realm_name_to_keyspace_name(nonempty_binary(), nonempty_binary()) :: nonempty_binary()
def realm_name_to_keyspace_name(realm_name, astarte_instance_id \\ "default") do
#...
```